index.html

```
@@ -19,7 +19,15 @@
       data-show-devmode-toggle-button="true"
       data-show-devmode-settings="true"
-    ></div>
+    >
+     <div id="loadingScreen">
+        <img id="loading-gif" src="assets/img/loadingImg.gif" alt="Loading Animation" />
+        <div id="progressBarContainer">
+          <div id="progressBar"></div>
+        </div>
+      </div>
+    </div>
+  </div>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </div> ]

(tag-pair)